preprocessing/text/cleaners.py

```diff
@@ -0,0 +1,128 @@
+import re
+
+from preprocessing.text.symbols import _alphabet, _punctuations, _not_end_punctuation, _numbers
+from preprocessing.text.numbers import Numbers
+
+
+class Cleaner:
+    def __init__(self, alphabet=None):
+        if not alphabet:
+            self.accepted_chars = list(_alphabet) + list(_punctuations) + list(_numbers)
+        self.abbreviations = {}
+        self.abbreviations_pattern = None
+    
+    def __call__(self, text):
```

Comment:
Add typehints. Here it is especially important since multiple types are accepted.